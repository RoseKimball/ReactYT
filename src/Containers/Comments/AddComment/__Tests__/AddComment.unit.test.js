import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import AddComment from '../AddComment';

describe('AddComment', () => {
    test('renders AddComment', () => {
        const wrapper = shallow (
            <AddComment />
        )
        expect(wrapper).toMatchSnapshot();
    })
}