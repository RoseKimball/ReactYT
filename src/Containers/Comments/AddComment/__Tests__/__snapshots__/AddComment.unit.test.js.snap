// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddComment renders AddComment 1`] = `ShallowWrapper {}`;
